core/src/main/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelector.java

```diff
@@ -83,13 +83,14 @@
  * certain scans. Second groups can be used to have different hardware/VM types for scans, for
  * example could have some scans use expensive high memory VMs and others use cheaper burstable
  * VMs.</li>
- * <li><b>enableTabletServerFallback : </b> When there are no scans servers, this setting determines
- * if fallback to tablet servers is desired. Falling back to tablet servers may cause tablets to be
- * loaded that are not currently loaded. When this setting is false and there are no scan servers,
- * it will wait for scan servers to be available. This setting avoids loading tablets on tablet
- * servers when scans servers are temporarily unavailable which could be caused by normal cluster
- * activity. If not specified this setting defaults to true. Set to false to avoid tablet server
- * fallback. Waiting for scan servers is done via
+ * <li><b>timeToWaitForScanServers : </b> When there are no scans servers, this setting determines
```

Comment:
Would be good to work in a suggested value for the case when someone wants to practiacally never fallback.  Could use `10000d` for this  as its 27 years.  Would also be good to add something about the supported time units for this value to the docs.  Looking at ConfigurationTypeHelper it supports ms,s,m,h,d units.

Reply:
The values do not have to numeric because we parse them. We could accept a timeout value of never, if its documented, and in the impl set that to max long.  I made this comment on the wrong thread a bit ago, duplicating it here.